examples/README.md

```diff
@@ -13,5 +13,6 @@ _In any of these cases, please leave a comment on the [Examples issue](https://g
 ## Structure
 
 - each directory contains an example `Dockerfile` and a `README.md`; ideally, it also has a `Dockerfile.before` showing the original, as created by the author, while the `Dockerfile` shows the improvements made based on the Ten Simple Rules
-- the `README.md` must include the source and license of the published Dockerfile, and the author; it should also include comments or notes, and a few keywords or a descriptive abstract that searching the repository gives reasonable results
+- the `README.md` must include the source and license of the published Dockerfile, and the author; it should also include comments or notes, a few keywords or a descriptive abstract that searching the repository gives reasonable results; it may include a (colourised) diff output, if the changes between Dockerfiles were small
```

Comment:
Have you ever seen a diff output in the wild?

Reply:
I don't get what you're implying here.

It's just a suggestion to give us and others and idea what could be in the README.

Reply:
Ah okay. I think it's a good suggestion, I just have never seen it.